benchmarks/src/main/scala/zio/IOBenchmarks.scala

```diff
@@ -15,6 +15,34 @@ object IOBenchmarks extends DefaultRuntime {
     override val Platform = PlatformLive.Benchmark.withTracing(Tracing.enabled)
   }
 
+  val SyncRuntime = new DefaultRuntime {
```

Comment:
We may want to move this into the `TestRuntimeBenchmarks.scala` file as these are the main benchmarks we use for comparing `ZIO` against other effect systems so we don't necessarily want to clutter them up with work we are doing to evaluate our test implementations.